extension/parquet/parquet_reader.cpp

```diff
@@ -147,6 +147,9 @@ static LogicalType DeriveLogicalType(const SchemaElement &s_ele) {
 				return LogicalType::BLOB;
 			}
 		}
+		if (binary_as_string && s_ele.converted_type == ConvertedType::UTF8) {
```

Comment:
Checking s_ele.converted_type is not a good idea if it is not set, since there is no guarantee it is set to anything valid (essentially it is garbage data). I would just default to `LogicalType::VARCHAR` here.